common/src/main/scala/explore/components/ui/ExploreStyles.scala

```diff
@@ -158,6 +158,7 @@ object ExploreStyles {
 
   val ItcPlotSection: Css = Css("itc-plot-section")
   val ItcPlotLoading: Css = Css("itc-plot-loading")
```

Comment:
`itc-plot-loading` was deleted from style.less. Does it have an functionality, anymore?